pkg/cosign/cosign_test.go

```diff
@@ -61,15 +69,15 @@ func TestCosignPayload(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Equal(t, d, "sha256:4a1c4b21597c1b4415bdbecb28a3296c6b5e23ca4f9feeb599860a1dac6a0108")
 
-	image2 := "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/nop"
-	signedPayloads2 := cosign.SignedPayload{Payload: []byte(tektonPayload)}
+	image2 := "ghcr.io/jimbugwadia/pause2"
```

Comment:
Same as above....best to use a Kyverno test image than rely on a user managed image.